run_redis.sh

```diff
@@ -0,0 +1,370 @@
+#!/bin/bash
+
```

Comment:
also, none of these scripts, if they are kept, should be in the top-level directory of this repo, please move them to a sane place under scripts for now, like `scripts/run-benchmarks/*`